packages/settings/api.d.ts

```diff
@@ -9,10 +9,36 @@ export interface CodeAssistSettings {
 
 export interface TraceSettings {
   trace: {
-    server: "off" | "messages" | "verbose";
+    server: keyof IValidTraceServerValues;
   };
 }
 
+export interface IValidTraceServerValues {
+  off: true;
+  messages: true;
+  verbose: true;
+}
+
+export const validTraceServerValues: IValidTraceServerValues;
+
+export interface LoggingSettings {
+  logging: {
+    level: "off" | "fatal" | "error" | "warn" | "info" | "debug" | "trace";
```

Comment:
Why not use `keyof IValidLoggingLevelValues` like you did in `TraceSettings`?

Reply:
copy / pasta mistake at 10pm 😄